### PR TITLE
css counter descriptors: prefix and suffix

### PR DESCRIPTION
**Pull request by estelle**
part of https://github.com/openwebdocs/project/issues/181

### CODE REVIEW

File: files/en-us/web/css/@counter-style/prefix/index.md [hunk 1]
```diff
@@ -7,21 +7,25 @@ browser-compat: css.at-rules.counter-style.prefix
...
-The **`prefix`** descriptor of the {{cssxref('@counter-style')}} rule specifies content that will be prepended to the marker representation. If not specified, the default value will be `""` (an empty string).
+The {{cssxref('@counter-style')}} rule's **`prefix`** descriptor specifies content that will be added to the beginning of the counter's marker representation.
+
+When the counter value is negative, the `prefix` comes before the negative sign and any other `<symbol>`s added by the {{cssxref("@counter-style/negative", "negative")}} descriptor.
 
 ## Syntax
 
 ```css
-/* <symbol> values */
+/* <symbol> values: string, image, or identifier */
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/prefix/index.md:17]: `<symbol> values` implies that multiple values can be added at the same time, but that's not the case according to the text description? In which case `<symbol> value` would be better?

File: files/en-us/web/css/@counter-style/prefix/index.md [hunk 2]
```diff
@@ -7,21 +7,25 @@ browser-compat: css.at-rules.counter-style.prefix
...
 prefix: "»";
 prefix: "Page ";
 prefix: url(bullet.png);
 ```
 
 ### Values
 
+The **`prefix`** descriptor takes as it's value a single `<symbol>`:
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/prefix/index.md:25]:
```suggestion
The **`prefix`** descriptor takes as its value a single `<symbol>`:
```

File: files/en-us/web/css/@counter-style/prefix/index.md [hunk 3]
```diff
@@ -7,21 +7,25 @@ browser-compat: css.at-rules.counter-style.prefix
...
+
 - `<symbol>`
-  - : Specifies a `<symbol>` that is prepended to the marker representation. It may be a {{cssxref("&lt;string&gt;")}}, {{cssxref("&lt;image&gt;")}}, or {{cssxref("&lt;custom-ident&gt;")}}.
+  - : Specifies a `<symbol>`, a {{cssxref("&lt;string&gt;")}}, {{cssxref("&lt;image&gt;")}}, or {{cssxref("&lt;custom-ident&gt;")}}, that is prepended to the marker representation.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/prefix/index.md:28]: I think this is now slightly less clear — it reads like `<symbol>` is the first item in the list of possibilities. To mitigate this issue, I would separate out the list with something more drastic, such as em-dashes.

```suggestion
  - : Specifies a `<symbol>` — a {{cssxref("&lt;string&gt;")}}, {{cssxref("&lt;image&gt;")}}, or {{cssxref("&lt;custom-ident&gt;")}} — that is prepended to the marker representation.
```

File: files/en-us/web/css/@counter-style/prefix/index.md [hunk 4]
```diff
@@ -35,29 +39,32 @@ prefix: url(bullet.png);
 
 ### Adding a prefix to a counter
 
+In this example, each counter number is prefixed by "Book " (with a space) and followed by a colon (`:`). The colon is added with the {{cssxref("@counter-style/suffix", "suffix")}} descriptor.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/prefix/index.md:42]: This example is really cool; I didn't know you could do this. I learned something today.

File: files/en-us/web/css/@counter-style/prefix/index.md [hunk 5]
```diff
@@ -76,5 +83,6 @@ prefix: url(bullet.png);
 
 ## See also
 
+- Other {{cssxref("@counter-style")}} descriptors, including, {{cssxref("@counter-style/system","system")}}, {{cssxref("@counter-style/symbols", "symbols")}}, {{cssxref("@counter-style/additive-symbols", "additive-symbols")}}, {{cssxref("@counter-style/negative", "negative")}}, {{cssxref("@counter-style/suffix", "suffix")}}, {{cssxref("@counter-style/range", "range")}}, {{cssxref("@counter-style/pad", "pad")}}, {{cssxref("@counter-style/speak-as", "speak-as")}}, and {{cssxref("@counter-style/fallback", "fallback")}}
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/prefix/index.md:86]:
```suggestion
- Other {{cssxref("@counter-style")}} descriptors: {{cssxref("@counter-style/system","system")}}, {{cssxref("@counter-style/symbols", "symbols")}}, {{cssxref("@counter-style/additive-symbols", "additive-symbols")}}, {{cssxref("@counter-style/negative", "negative")}}, {{cssxref("@counter-style/suffix", "suffix")}}, {{cssxref("@counter-style/range", "range")}}, {{cssxref("@counter-style/pad", "pad")}}, {{cssxref("@counter-style/speak-as", "speak-as")}}, and {{cssxref("@counter-style/fallback", "fallback")}}
```

File: files/en-us/web/css/@counter-style/prefix/index.md [hunk 6]
```diff
@@ -76,5 +83,6 @@ prefix: url(bullet.png);
...
 - {{Cssxref("list-style")}}, {{Cssxref("list-style-image")}}, {{Cssxref("list-style-position")}}
 - {{cssxref("symbols", "symbols()")}}, the functional notation creating anonymous counter styles
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/prefix/index.md:88]:
```suggestion
- {{cssxref("symbols", "symbols()")}}: the functional notation for creating anonymous counter styles
```

File: files/en-us/web/css/@counter-style/suffix/index.md [hunk 7]
```diff
@@ -7,19 +7,21 @@ browser-compat: css.at-rules.counter-style.suffix
 
 {{CSSRef}}
 
-The **`suffix`** descriptor of the {{cssxref("@counter-style")}} rule specifies content that will be appended to the marker representation.
+The **`suffix`** descriptor of the {{cssxref("@counter-style")}} rule specifies content that will be appended at the end of the marker representation.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/suffix/index.md:10]: "appended" means "added to the end of", so this edit is not really needed. You could just follow the same style as you've used on the `prefix` page, and say "...that will be added to the end of..."

File: files/en-us/web/css/@counter-style/suffix/index.md [hunk 8]
```diff
@@ -7,19 +7,21 @@ browser-compat: css.at-rules.counter-style.suffix
...
 
 ## Syntax
 
 ```css
-/* <symbol> values */
+/* <symbol> values: string, image, or identifier  */
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/suffix/index.md:15]: Same comment as for the `prefix` page. Would `<symbol> value` be better?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle fine work! I've left you some suggestions and comments.

File: files/en-us/web/css/@counter-style/suffix/index.md (hunk 7 above)
Comment (estelle) [reply files/en-us/web/css/@counter-style/suffix/index.md:10]: I added it for non-native English speakers, but changing to added is even better!

Comment (estelle) [inline files/en-us/web/css/@counter-style/suffix/index.md:10]:
```suggestion
The **`suffix`** descriptor of the {{cssxref("@counter-style")}} rule specifies content that will be added to the end of the marker representation.
```

File: files/en-us/web/css/@counter-style/prefix/index.md (hunk 1 above)
Comment (estelle) [inline files/en-us/web/css/@counter-style/prefix/index.md:17]:
```suggestion
/* <symbol> value: string, image, or identifier */
```

File: files/en-us/web/css/@counter-style/suffix/index.md (hunk 8 above)
Comment (estelle) [inline files/en-us/web/css/@counter-style/suffix/index.md:15]:
```suggestion
/* <symbol> value: string, image, or identifier  */
```

Comment (estelle) [reply files/en-us/web/css/@counter-style/suffix/index.md]: dropped the "s"

**Comment by estelle** [comment]
Thanks @chrisdavidmills !

**Review by chrisdavidmills — APPROVED** [review]
Yusss, this is perfect, great work @estelle! Let's get this merged.

**Pull request merged**